server/fileserver.go

```diff
@@ -101,7 +101,13 @@ func (fh *fileHandler) serveFile(w http.ResponseWriter, r *http.Request, name st
 	// Still a directory? (we didn't find an index file)
 	// Return 404 to hide the fact that the folder exists
 	if d.IsDir() {
-		return http.StatusNotFound, nil
+		if name == "/" {
```

Comment:
You should not expose the webserver's name like this. You should have a default configuration with this contents (as other webservers do)